README.md

```diff
@@ -1,44 +1,47 @@
-<p align="center"><h1 align="center">
```

Comment:
@darkwizard242 it looks like you deleted some things that shouldn't have been deleted
can you update the PR to restore the original READMe format and just add the Docker section that you have?

Reply:
@lirantal  - Ah sorry, that seems to have been due to the Markdown plugin on the Atom text editor I was using.  I will add correct the README and push the changes to the PR now.

Reply:
Cool thanks. I expected it was probably some auto-formatter :)

Reply:
Yeah, that's my bad. Sorry about that. I used vim this time lol.

Corrected and pushed the README back now 👍